support/sparql-queries.js

```diff
@@ -589,6 +602,21 @@ async function insertDocuments(kaleidosPieces, agendaitem, graph) {
           nfo:fileName ?fileNamePhysicalFile ;
           nfo:fileSize ?sizePhysicalFile ;
           dbpedia:fileExtension ?extensionPhysicalFile .
+        OPTIONAL {
```

Comment:
I'm a bit afraid, based on previous experience with Virtuoso, that the `OPTIONAL` will have a major impact on the query performance. Can you split of in a separate query?